src/contract/tokens.rs

```diff
@@ -147,13 +147,13 @@ impl Tokenizable for H256 {
 impl Tokenizable for Address {
   fn from_token(token: Token) -> Result<Self, Error> {
     match token {
-      Token::Address(data) => Ok(types::H160(data)),
+      Token::Address(data) => Ok(data.into()),
```

Comment:
in `ethabi 5.1` `data` is already an `Address`, there is no need to call `.into()`, [docs](https://docs.rs/ethabi/5.1.0/ethabi/enum.Token.html#variant.Address)